### ISSUE TITLE
Type Synonyms not exported to top-level module

### ISSUE THREAD

**Issue by jcampbell** [snowflakedb/snowflake-sqlalchemy#109]
Please answer these questions before submitting your issue. Thanks!

1. What version of Python are you using (`python --version`)?
Python 3.7.3

2. What operating system and processor architecture are you using (`python -c 'import platform; print(platform.platform())'`)?
Darwin-18.7.0-x86_64-i386-64bit

3. What are the component versions in the environment (`pip list`)? 
```
...
snowflake-connector-python    1.8.5         
snowflake-sqlalchemy          1.1.13        
SQLAlchemy                    1.3.6  
...
```

4. What did you do?
If possible, provide a recipe for reproducing the error.
A complete runnable program is good.

```
import snowflake.sqlalchemy.snowdialect
getattr(snowflake.sqlalchemy.snowdialect, "VARCHAR")  # correctly returns type
getattr(snowflake.sqlalchemy.snowdialect, "TEXT")  # raises AttributeError; fails to reflect snowflake-supported type synonym
```

5. What did you expect to see?
I expected both statements to return `sqlalchemy.sql.sqltypes.VARCHAR`

6. What did you see instead?
I see an AttributeError

7. Can you set logging to DEBUG and collect the logs?

```
import logging
import os

for logger_name in ['snowflake.sqlalchemy', 'snowflake.connector', 'botocore']: 
    logger = logging.getLogger(logger_name)
    logger.setLevel(logging.DEBUG)
    ch = logging.StreamHandler()
    ch.setLevel(logging.DEBUG)
    ch.setFormatter(logging.Formatter('%(asctime)s - %(threadName)s %(filename)s:%(lineno)d - %(funcName)s() - %(levelname)s - %(message)s'))
    logger.addHandler(ch)
```
Yes; the only logging produced is during the import itself:
```
2019-07-25 17:27:51,642 - MainThread ssl_wrap_socket.py:86 - inject_into_urllib3() - DEBUG - Injecting ssl_wrap_socket_with_ocsp
2019-07-25 17:27:51,654 - MainThread auth.py:66 - <module>() - DEBUG - cache directory: /Users/james/Library/Caches/Snowflake
```

snowdialect.py apears to use the `ischema_names` dict internally for type mapping, but the sqlalchemy docs and norm is to export those items as top-level names in the dialect (see, e.g. postgres docs (recognize it's a "supported" and "included" dialect): 

> As with all SQLAlchemy dialects, all UPPERCASE types that are known to be valid with PostgreSQL are importable from the top level dialect, whether they originate from sqlalchemy.types or from the local dialect:

**Comment by smtakeda** [comment]
If the types should be exported at top level, wouldn't the test cases be like this?
```
import snowflake.sqlalchemy
getattr(snowflake.sqlalchemy, "VARCHAR")   # returning sqlalchemy.sql.sqltypes.VARCHAR
getattr(snowflake.sqlalchemy, "TEXT")  # returning sqlalchemy.sql.sqltypes.VARCHAR
```

**Comment by jcampbell** [comment]
I believe that's correct, yes.

**Comment by smtakeda** [comment]
Fixed in 1.1.15

**Comment by jcampbell** [comment]
Thank you @smtakeda !
